### PR TITLE
Make use of a '.env' file to specify a default version of StackStorm in docker-compose

### PR DESCRIPTION
**Pull request by jvrplmlmn**
Solves https://github.com/StackStorm/st2-dockerfiles/issues/34

Includes https://github.com/StackStorm/st2-dockerfiles/pull/33, but I can rebase this to keep it a single issue solved per PR. EDIT: This is right now out of sync with #33 

WDYT @armab @emedvedev ?

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MONGO_VERSION=3.2
```
Comment (armab) [inline .env]: We don't want to expose mongo version decision to users. Eg. it's only and only 3.2
So it's better to hardcode it in `docker-compose.yml`

**Comment by armab** [comment]
So besides of the above comments, this is useful change.
Especially when we start promoting latest Docker images to hub (we stopped doing that).

**Comment by jvrplmlmn** [comment]
Since this includes #33, this is blocked until CircleCI allows greater versions of Docker, see https://github.com/StackStorm/st2-dockerfiles/pull/33#issuecomment-264723302 for reference.

**Comment by armab** [comment]
Ah, `.env` file is available only with compose `2` schema, or we can just exclude https://github.com/StackStorm/st2-dockerfiles/pull/35/commits/b1320b5b26a5c0b9d07c1761d71b4e4ef8682e42 from this PR commit history and merge `env` change only?

**Comment by jvrplmlmn** [comment]
I can remove #33 and see how the CI behaves, but on my laptop running `docker-compose config` will render `version: 2.0`, so not sure what will happen.

**Comment by armab** [comment]
Closed by https://github.com/StackStorm/st2packaging-dockerfiles/pull/68

**Pull request closed without merging**